packages/@expo/cli/src/export/index.ts

```diff
@@ -53,6 +54,7 @@ export const expoExport: Command = async (argv) => {
         chalk`-p, --platform <platform>  Options: android, ios, web, all. {dim Default: all}`,
         `-s, --source-maps          Emit JavaScript source maps`,
         `-c, --clear                Clear the bundler cache`,
+        `-v, --verify-assets <path> Given a path to the build's embedded manifest, check for assets omitted from the export (android, ios)`,
```

Comment:
Not in love with adding a native-only flag for a command that bundles javascript universally. What would need to change in the browser to make this feature supported on web? Could we change the native implementation to support web?

On web, you don't may not need a feature like this because assets are fetched in different requests, unless they're inlined in the JS bundle. In the case of emitting bundler artifacts conditionally based on runtimes, my mind goes to "externals". Externals are imports that you leave as-is because you're bundling for a target that supports the import syntax that's being used. Perhaps the asset equivalent is a convention that indicates to the bundler that an asset is built-in to the native runtime. In this case, I think about the developer tooling that's built around Node.js built-ins, things like TypeScript types for the environment and lint presets that indicate what the target environment should look like.

Reply:
@EvanBacon would it be better to implement this as a separate command like `export:verify-assets`?

Reply:
On the idea of "externals": the concern I have would be that customers who have thousands of assets in their app would have to modify all their import statements to follow the new convention.

I think the right way to use this feature is to make sure that all built-in assets are in a specific directory structure (like `embeddedAssets`), and any asset that needs to be added or changed would be added under a single directory that matches `extra.updates.assetPatternsToBeBundled`.